data/preprocessing/livestock_processed/Makefile

```diff
@@ -0,0 +1,100 @@
+		-A $(data_dir)/pasture/6_Ct_2010_Aw.tif \
+		-B $(data_dir)/pasture/6_Gt_2010_Aw.tif \
+		-C $(data_dir)/pasture/6_Sh_2010_Aw.tif \
+		--outfile $(data_dir)/pasture/6_Tm_2010_Aw.tif
+
+#rasterise and calculate the tonnes of material
+rasterize_and_calculate_commodities_production:
+	mkdir -p $(data_dir)/rasterized/production
+	mkdir -p $(data_dir)/processed_commodities/production
+	gdal_rasterize -q -l FAOSTAT_data_hens_eggs_iso3_2021_t -a Value -tr 0.083333 0.083333 -a_nodata 0 \
+		-te -180.0 -89.99928 179.99856 90.0 -ot Float32 -of GTiff \
+		$(data_dir)/faostats_processed/production/FAOSTAT_data_hens_eggs_iso3_2021_t.shp \
+		$(data_dir)/rasterized/production/FAOSTAT_data_hens_eggs_iso3_2021_t.tif;
+	gdal_calc.py --quiet --calc "A*B" --format GTiff --type Float32 --NoDataValue 0.0 \
+		-A $(data_dir)/rasterized/production/FAOSTAT_data_hens_eggs_iso3_2021_t.tif --A_band 1 \
+		-B $(data_dir)/pasture/6_Ch_2010_Aw.tif \
+		--outfile $(data_dir)/processed_commodities/production/GLO_2021_HensEggs_t.tif;
+	gdal_rasterize -q -l FAOSTAT_data_total_milk_iso3_2021_t -a Value -tr 0.083333 0.083333 -a_nodata 0 \
+		-te -180.0 -89.99928 179.99856 90.0 -ot Float32 -of GTiff \
+		$(data_dir)/faostats_processed/production/FAOSTAT_data_total_milk_iso3_2021_t.shp \
+		$(data_dir)/rasterized/production/FAOSTAT_data_total_milk_iso3_2021_t.tif;
+	gdal_calc.py --NoDataValue=0 --quiet --calc "A*(B!=3.40282e+38)*B" --format GTiff --type Float64 \
+		-A $(data_dir)/rasterized/production/FAOSTAT_data_total_milk_iso3_2021_t.tif --A_band 1 \
+		-B $(data_dir)/pasture/6_Tm_2010_Aw.tif \
+		--outfile $(data_dir)/processed_commodities/production/GLO_2021_TotalRawMilk_t.tif;
+
+rasterize_and_calculate_commodities_harvest:
+	mkdir -p $(data_dir)/rasterized/harvest
+	mkdir -p $(data_dir)/processed_commodities/harvest
+	gdal_rasterize -q -l FAOSTAT_data_hens_eggs_iso3_2021_ha -a Value -tr 0.083333 0.083333 -a_nodata 0 \
+		-te -180.0 -89.99928 179.99856 90.0 -ot Float32 -of GTiff \
+		$(data_dir)/faostats_processed/harvest/FAOSTAT_data_hens_eggs_iso3_2021_ha.shp \
+		$(data_dir)/rasterized/harvest/FAOSTAT_data_hens_eggs_iso3_2021_ha.tif;
+	gdal_calc.py --quiet --calc "B/((A!=0)*A)" --format GTiff --type Float32 --NoDataValue 0.0 \
+		-A $(data_dir)/rasterized/harvest/FAOSTAT_data_hens_eggs_iso3_2021_ha.tif --A_band 1 \
+		-B $(data_dir)/pasture/6_Ch_2010_Aw.tif \
+		--outfile $(data_dir)/processed_commodities/harvest/GLO_2021_HensEggs_ha.tif;
+	gdal_rasterize -q -l FAOSTAT_data_total_milk_iso3_2021_ha -a Value -tr 0.083333 0.083333 -a_nodata 0 \
+		-te -180.0 -89.99928 179.99856 90.0 -ot Float32 -of GTiff \
+		$(data_dir)/faostats_processed/harvest/FAOSTAT_data_total_milk_iso3_2021_ha.shp \
+		$(data_dir)/rasterized/harvest/FAOSTAT_data_total_milk_iso3_2021_ha.tif;
+	gdal_calc.py --NoDataValue=0 --quiet --calc "((B!=3.40282e+38)*B)/((A!=0)*A)" --format GTiff --type Float64 \
```

Comment:
same for this number that should have a comment or something explaining what it is doing